WhenPushComesToShove/Assets/3-Scripts/Utilities/Physics/VelocitySetter.cs

```diff
@@ -1,19 +1,20 @@
 using System.Collections;
 using System.Collections.Generic;
 using UnityEngine;
+using DG.Tweening;
 
 
 //direction and speed stored seperately so that speed can be tweened
 public struct VelocityData
 {
     public Vector2 direction;
-    public float speed;
+    public float  speed;
```

Comment:
Extra space here thats not needed

Reply:
Ah thank you, the entire struct is actually not needed